### PR TITLE
Feat/xd 110

### PR DESCRIPTION
**Pull request by Persists**

### CODE REVIEW

**Comment by Persists** [comment]
I know that there are still service tests missing, and the seeding may not be correct, using the enpoint

File: libs/common/backend/insight-hub/src/lib/services/iot-time-series.service.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export class XdIotTimeSeriesService extends XdBaseBearerInteractionService {
 	 * Allows to get the time series data from the IoT Time Series API.
 	 * @see https://documentation.mindsphere.io/MindSphere/apis/iot-iottimeseries/api-iottimeseries-api.html
 	 *
-	 * @param assetId The id of the asset entity
+	 * @param assetId The id of the asset asset
```
Comment (KonsumGandalf) [inline libs/common/backend/insight-hub/src/lib/services/iot-time-series.service.ts]: Maybe this is wrong

File: libs/common/backend/insight-hub/src/lib/services/users.service.ts [hunk 2]
```diff
@@ -27,6 +27,8 @@ export class XdUsersService extends XdBaseBearerInteractionService {
 	 * Allows to get the users data from the User API.
 	 */
 	public getUsersData(): Observable<IUsersResponse> {
-		return super._getData<IUsersResponse>();
+		return super._getData<IUsersResponse>(
+
```
Comment (KonsumGandalf) [inline libs/common/backend/insight-hub/src/lib/services/users.service.ts]: why this line?

**Review by KonsumGandalf — CHANGES_REQUESTED** [review]
Great PR 2 small remarks

File: libs/common/backend/insight-hub/src/lib/services/iot-time-series.service.ts (hunk 1 above)
Comment (Persists) [reply libs/common/backend/insight-hub/src/lib/services/iot-time-series.service.ts]: true

**Review by KonsumGandalf — APPROVED** [review]
LGTM

**Pull request merged**